index.html

```diff
@@ -1,13 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>"ALL about me"</title>
+    <meta name="Farida" content="HTML"/>
+  </head>
+  <body>
 <h1>Farida Hyderi</h1>
+    <section id="About">
 <h2>About</h2>
 <p>My name is Farida I am very entusiastic to learn new programming langusges
 also very much interested in testing </p>
+    </section>
+
+    <section id = "Experience">
 <h2>Experience</h2>
-Cognizant Agile Methodology from Forage
-Accenture Project management Forage
-Mastercard - Cybersecurity Job Simulation from Forage
-Worked on testing projects
+      <ul>
+<li>Cognizant Agile Methodology from Forage </li>
+<li>Accenture Project management Forage </li>
+<li>Mastercard - Cybersecurity Job Simulation from Forage</li>
+<li>Worked on testing projects</li>
+      </ul>
+    </section>
+
+    <section id = "connect">
 <h2>Connect</h2>
-<li><www.linkedin.com/in/farida-hyderi-a32700226</www.linkedin.com</li>
-<li>gmail.com</li>
-<li>instagram</li>
+<a href="www.linkedin.com/in/farida-hyderi-a32700226</www.linkedin.com"> My linkedin </a>
```

Comment:
Well done with fixing the format of these links. This probably should be:
`<a href="www.linkedin.com/in/farida-hyderi-a32700226"> My LinkedIn </a>` so it properly to your LinkedIn. On a personal note, I've had potential employers check out my GitHub many times before/after interviews, so you'll want this to be accurate.